setup.py

```diff
@@ -14,19 +14,25 @@
     "accept-rom-license": ["autorom[accept-rom-license]~=0.4.2"],
     "box2d": ["box2d-py==2.3.5", "pygame==2.1.0"],
     "classic_control": ["pygame==2.1.0"],
-    "mujoco": ["mujoco_py>=1.50, <2.0"],
+    "mujoco_py": ["mujoco_py<2.2,>=2.1"],
```

Comment:
I have mild concerns around `gym` declaring a dependency on both `mujoco` and `mujoco-py`. In principle, this isn't a problem as long as only one gets imported, but if someone tries to import both it's likely that they'll run into inscrutable errors. Would you consider declaring `mujoco-py` as an optional dependency (`extras_require`)?

Reply:
Ignore me: this is _already_ in `extras_require`.